### DISCUSSION TITLE
Analysis Question

### DISCUSSION THREAD

**Discussion by dlcn1994** [G-String-Legacy/G_String#24] [General]
I am just hoping to seek some help on setting up my G-string analysis as I am struggling despite reviewing existing literature (i'm a surgeon attempting to conduct this analysis with limited mathematical background).
 
I have surgical trainees (t) being assessed performing a single procedure. 
Assessment is undertaken using an assessment tool which assesses 4 domains of competence or items (i). 
Each procedure is assessed by two assessors (a). 

This gives me t x i x a, I believe my facet of differentiation is the trainees (t). 

However, the assessors are not the same for every trainee. We have a pool of say 10 assessors, and each trainee will be assessed by a random 2 assessors. 

How do I factor this into the G-string software and my data setup?

Kind regards,

Dan

**Comment by Papa-26** [comment]
Hello Dan,
You happened to ask your question on the old legacy site, it is about 3 years out of date. The current version is: [GS_MV](https://github.com/G-String-Legacy/GS_MV). I suggest, you go there and and go to the [wiki](https://github.com/G-String-Legacy/GS_MV/wiki) . You find there a lot of instructional material. You don't need any mathematical expertise. Look at G-String as a mental "prosthesis".  
There is, however, a problem with your design: I don't assume that your patients are patient enough to allow the same procedure be performed on them for each trainee ;<\. In other words, the evaluation task is nested in trainee. And since each candidate is evaluated only on one unique procedure, your evaluation won't be valid. That would be different, if each trainee were to be on a sufficiently large random sample of procedures, or patients.
If you want to discuss this further, do it on https://github.com/G-String-Legacy/GS_MV.

Ralph

**Comment by dlcn1994** [comment]
Dear Ralph,

Thank you for getting back to me so quickly, and for kindly directing to
the correct site, I will review that for information.

It is actually a simulated assessment, so the same procedure replicated
every time! With a standardised assessment tool, and a pool of 16
assessors. I can post the question again in the legacy site you provided.

Kind regards,

Daniel
